src/content/docs/dropins/checkout/containers/overview.mdx

```diff
@@ -46,3 +46,91 @@ The `Title` property is a handler used to render a customized title content.
 :::note[Checkout containers]
 In the pages that follow, you will learn how to customize the `Title` and how to use it.
 :::
+
+## Active property
+
+The `active` property controls whether the container is active, this means it is visible and responds to system events. When set to _false_, the container is not reactive, therefore it does not subscribe to system events and it's not rendered in the user interface.
+
+### `active` prop declaration
+
+<OptionsTable
+  compact
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['active', 'boolean', 'No', 'Activates/deactivates the container (default value is true).'],
+  ]}
+/>
+
+```ts
+active?: boolean;
+```
+
+#### Behavior:
+
+- When `active` is _true_:
+  - The container subscribes to system events
+  - It renders in the user interface and it is visible (shows a skeleton or renders the component in the UI)
+  - It executes its normal operations
+- When `active` is _false_:
+  - The container does not subscribe to system events
+  - It does not render in the user interface and its visibility is set to false
+  - Side effects and state updates are not executed
+
+### Example of `active` prop definition
+
+In the following example, the ShippingMethods container will be deactivated; this means it won't be rendered to the UI and won't listen to system events.
```

Comment:
```suggestion
In the following example, the `ShippingMethods` container will be deactivated; this means it will not be rendered to the UI and will not listen to system events.
```